www/runtest.php

```diff
@@ -2235,8 +2235,8 @@ function CreateTest(&$test, $url, $batch = 0, $batch_locations = 0)
                 AddIniLine($testFile, 'lighthouseTrace', '1');
             if( isset($test['lighthouseThrottle']) && $test['lighthouseThrottle'] )
                 AddIniLine($testFile, 'lighthouseThrottle', '1');
-            if( isset($test['heroElementTimes']) && $test['heroElementTimes'] )
-                AddIniLine($testFile, 'heroElementTimes', '1');
+            if( isset($test['heroElementTimes']) )

```

Comment:
No need to change this if the agent continues to have it disabled by default and have the system default controlled by settings.ini